book/9-begin/server/stripe.js

```diff
@@ -7,8 +7,7 @@ const User = require('./models/User');
 const dev = process.env.NODE_ENV !== 'production';
 const API_KEY = dev ? process.env.STRIPE_TEST_SECRETKEY : process.env.STRIPE_LIVE_SECRETKEY;
 
-const port = process.env.PORT || 8000;
-const ROOT_URL = `http://localhost:${port}`;
+const ROOT_URL = require('../lib/api/getRootUrl');
```

Comment:
I'm actually not sure if this would work, I would think you have to call the function with a set of parenthesis. 
```
const getRootUrl = require('../lib/api/getRootUrl');
const ROOT_URL = getRootUrl();
```